packages/core/docs/contributing/themes.md

```diff
@@ -10,18 +10,28 @@ Default theme is located in `packages/core/theme-module` folder and recognized a
 
 ## Configuration
 
-To inherit from the default theme in your integration theme you need to install private `@vue-storefront/nuxt-theme` package. from this repo.
+Run VSF CLI create project command with 2 arguments. The first is name of the integration, e.g. `commercetools`, `about-you` or just `boilerplate`. Latter is the target path where your project will be created.
```

Comment:
There is no such command yet. Also this document is meant to explana how themes are working not how to create a project and... You deleted that part :d